instrumentation/opentelemetry-api-0.4/src/main/java/io/opentelemetry/auto/instrumentation/opentelemetryapi/trace/Bridging.java

```diff
@@ -163,23 +153,23 @@ public static SpanContext toUnshaded(final io.opentelemetry.trace.SpanContext sh
         .build();
   }
 
-  private static TraceId toUnshaded(final io.opentelemetry.trace.TraceId shadedTraceId) {
+  public static TraceId toUnshaded(final io.opentelemetry.trace.TraceId shadedTraceId) {
     final byte[] bytes = getBuffer();
     shadedTraceId.copyBytesTo(bytes, 0);
     return TraceId.fromBytes(bytes, 0);
```

Comment:
Ouch. Since this is an agent is it possible to inject bytecode to expose an accessor to the byte array so it can be shared?